services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -97,6 +101,18 @@ func Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+
+	var eslProcessingBackoff uint64
+	if val, exists := os.LookupEnv("KUBERPULT_ESL_PROCESSING_BACKOFF"); !exists {
```

Comment:
Why are we doing this in a different manner than previously? Not saying any individual one is better, but maybe we should strive for consistency